spec/models/user_spec.rb

```diff
@@ -1,9 +1,19 @@
 require 'spec_helper'
 
 describe User do
-  it { expect(subject).to validate_presence_of(:name) }
+  it { expect(subject).to validate_presence_of(:nickname) }
   it { expect(subject).to validate_presence_of(:home_city_id) }
 
+  context 'names' do
+    let(:user) { create(:user) }
+
+    describe '.name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.